t/24-worker.t

```diff
@@ -208,6 +209,88 @@ subtest 'mock test stop_job' => sub {
     is $stop_job, 1, "stop_job() reached";
 };
 
+subtest 'worker configuration reading' => sub {
+    use Mojo::File qw(tempdir path);
+    my $configdir = tempdir();
+    local $ENV{OPENQA_CONFIG} = $configdir;
+    my $ini = $configdir->child('workers.ini');
+    $ini->spurt(
+        <<EOF
+# Configuration of the workers and their backends.
+[global]
+CACHEDIRECTORY = /var/lib/openqa/cache
+HOST = localhost foobar
+TOTAL_INSTANCES = 6
```

Comment:
This shouldn't be required - at all. We manage the number of workers in systemd by the admin creating pool directories.

Reply:
It's not required - it's just supported, population is sent from webui to the worker instance

Reply:
https://github.com/os-autoinst/openQA/pull/1486/files#diff-4234193dc63be2e7aee5593a65382621R265